docs/specs/authentication-flow.md

```diff
@@ -275,34 +275,6 @@ login_flows:
   # If the condition does not hold, this step is no-op.
   - type: change_password
     target_step: step1
-
```

Comment:
We still have some account linking content in this document. Please remove them. Also add a backlink to account-linking.md to tell the reader to find the new document.